l10n_nl_cbs_export/__manifest__.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# See LICENSE file for full copyright and licensing details.
+
+{
+    "name": "CBS Export Intrahandel Sale",
+    "version": "1.0",
+    "author": "Odoo Experts",
+    "website": "https://www.odooexperts.nl",
+    "category": "Accounting",
+    "depends": ["account", "report_intrastat"],
+    "summary": "CBS Export File for Dutch Intrahandel Sale",
+    "description": """
+        CBS Export File for Dutch Intrahandel Sale
+        
+        See also: https://www.cbs.nl/nl-nl/deelnemers-enquetes/deelnemers-enquetes/bedrijven/onderzoek/lopend/internationale-handel-in-goederen/idep-codelijsten
+        
+    """,
```

Comment:
remove this and use a [README.rst](https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst)